cogs/study.py

```diff
@@ -0,0 +1,80 @@
+import discord
+from discord.ext.commands import Bot, Cog, Context, command
+import aiohttp
+
+from utils.mathscrape import get_math_results
+from utils.wolframscrape import get_wolfram_data
+from .urbandict_utils.urbandictpages import UrbanDictionaryPages
+
+
+class Study(Cog):
+    def __init__(self, bot) -> None:
+        self.bot = bot
+
+    @command()
+    async def pcalc(self, ctx: Context) -> None:
+        pass
```

Comment:
```suggestion
```
Empty commands shouldn't be here, not even as a reminder to do it, you can mark that either using `# TODO: text` or outside of the project.